package.json

```diff
@@ -4,12 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "start": "npx babel-node src/index.js",
+    "build": "babel -d build src",
+    "start": "npm run build && node build/index.js",
     "lint": "eslint .",
-    "test-server": "NODE_ENV=test babel-node src/index.js",
-    "test": "NODE_ENV=test nyc jest -w 1 --detectOpenHandles --forceExit",
-    "posttest": "nyc report --reporter=json",
+    "test": "jest",
+    "test:coverage": "jest --coverage",
```

Comment:
the parameter on jest `-w 1` and was to have success test on `circleci` because it only provide single worker. I know this is `deprecated` but this flag actually passed me build successfully in `circleci`.

Also `--detectOpenHandles` was used to finish up test after `test` is done in `testServer`.

will using `jest` would be find?

Reply:
yes just using `jest` is fine for most cases. if we handle all promise rejections in the right way.

Reply:
Ok we will see this!